### PR TITLE
Added <PageTemplate>

### PR DESCRIPTION
**Pull request by LamboCreeper**
Moved over to using `<PageTemplate>` for consistency, will also allow us to create something like `<PrivatePageTemplate>` for pages that require login without us re-producing code. Should make everything more DRY in the long run.

### CODE REVIEW

File: src/components/molecules/Navigation.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ class Navigation extends Component {
                         </ul>
                         <ul className="nav-right">
                             <li onClick={this.toggleLoginModal}>
-                                <Link to="#">
+                                <Link href="#">
```
Comment (Jason2605) [inline src/components/molecules/Navigation.js]: Potential oversight?

File: src/components/templates/ErrorTemplate.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ function ErrorTemplate({message, code}) {
                 <p>
                     {message}
                 </p>
-                <Link to="/">
+                <Link href="/">
```
Comment (Jason2605) [inline src/components/templates/ErrorTemplate.js]: Potential oversight?

File: src/components/molecules/Navigation.js (hunk 1 above)
Comment (LamboCreeper) [reply src/components/molecules/Navigation.js]: Yep, good spot!

File: src/components/templates/ErrorTemplate.js (hunk 2 above)
Comment (LamboCreeper) [reply src/components/templates/ErrorTemplate.js]: Yep, good spot!

**Review by Jason2605 — APPROVED** [review]

**Pull request merged**